AstSemantics.md

```diff
@@ -154,6 +151,28 @@ by [available resources](Nondeterminism.md)). In the future, to support
 [>4GiB linear memory](FutureFeatures.md#heaps-bigger-than-4gib), support for
 indices with type `int64` will be added.
 
+### Adjusting memory size and permissions: `mmap`
+
+In the MVP the size of linear memory is fixed: The initial size of linear
+memory will remain unchanged for the life of that WebAssembly module. Later,
```

Comment:
Well, the wasm binary is going to have a data segment, and that has to be written in after memory is set up. It seems like that means that if the user has manual control of `mmap` for the initial size of memory, that they would also need to have manual control of copying the data into memory, so that they can do so afterwards. Do we actually want that?